src/main/java/com/example/gradecalculator/entities/Subject.java

```diff
@@ -1,25 +1,34 @@
 package com.example.gradecalculator.entities;
 
 import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
 
+import java.util.Set;
+
+@Setter
+@Getter
 @Entity
 @Table(name = "subject")
 public class Subject {
 
+    @Setter
```

Comment:
I think david said before when it is done on entity level you do not need it here? check all files for that maybe? it was resolved here but does not seem to be done